src/components/map/Map.scss

```diff
@@ -0,0 +1,24 @@
+// *{
+//     border: 2px solid red;
+// }
+
+main{
+   
+    position:relative;
+}
+
+.map img{
+    padding: 5%;
+    width:90%;
+    height: auto;
```

Comment:
`height: auto` is pretty bold. Do you know exactly what it's doing here, and will it work on all screen sizes?